### ISSUE TITLE
Block all javascript?

### ISSUE THREAD

**Issue by tmpm697** [jun7/wyebadblock#6]
like title, I find this adblock still bypass by some ads, I want same behavior of `noscript` on firefox or `ublock origin`, but if it's hard, just block all javascripts. truly it's hard to use a browser without an efficient adblocker.

**Comment by jun7** [comment]
Ah are you using wyeb? It is on wyeb.

**Comment by tmpm697** [comment]
Yes, I'm using `wyeb`, can you give an example in readme to give best practice for this plugin to use with `wyeb`?
